### DISCUSSION TITLE
Antidote Tox app on Apple app store

### DISCUSSION THREAD

**Discussion by supersophie** [community/community#108905] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

We depend on Antidote for our comms.

We have Antidote installed on our iphones ( 7 x iphones running ios 15.), and two old iPads running ios 12.

I had to set up two iphones for another member, but saw that Antidote had been removed from the App Store ( UK App Store , AU App Store). 

The message was " is not available in your region".

This was a real blow, because we don't want to move back to WhatsApp. Element/Matrix is an option but still not ideal.

We would be very grateful if you could you make your Antidote available on the app store for UK and AU regions ( United Kingdom and Australia).    Do you think you could?

Looking forward to a positive reply.

Regards,
S.

**Comment by queenofcorgis** [comment]
Hi @supersophie ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Antidote. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by zoff99** [comment]
@supersophie antidote is available on the appstore but its source code is not published anywhere.
